docs/content/python/new-roles/examples.md

```diff
@@ -38,8 +38,9 @@ This defines two roles on repository, `"reader"` and `"writer"`.
 This allows us to assign users directly to repositories without
 giving them a role in the entire organization.
 
-Now, we can use `OsoRoles.assign_role` to assign role on
-repositories in addition to organizations.
+Now, we can use
+{{% apiDeepLink "OsoRoles" "OsoRoles.assign_role" "sqlalchemy" %}}assign_role{{% /apiDeepLink %}}
```

Comment:
This seems like the only example where we're passing more than one param to `apiDeepLink`, but I'm not totally sure why we need the middle param (`$label`) here. Won't `$genText` be equal to `OsoRoles.assign_role` as well?

Could you provide an example of where we would need to override `$genText` with `$label`?

Reply:
There are some cases in the sqlalchemy and django documentation where we need to differentiate which package something is coming from (eg `oso.Oso.foo` vs `sqlalchemy_oso.SqlalchemyOso.foo`) and I wanted to add a manual override for those cases

Reply:
Okay I think I follow, but would you mind doing one of those changes on this PR just so I can see what it looks like in the wild?